.azure-pipelines/client.yml

```diff
@@ -48,11 +48,14 @@ jobs:
 
```

Comment:
For consistency with our publish guidelines could you name the artifact "packages" similar to what we did in the JS repo https://github.com/Azure/azure-sdk-for-js/blob/master/.azure-pipelines/client.yml#L52-L56 so that the release pipelines can work the same.

Reply:
Done